components/contribute-cards/EditTierModal.js

```diff
@@ -801,6 +828,76 @@ export function EditTierForm({ tier, collective, onClose }) {
           );
         }}
       </Formik>
+      {isConfirmingDelete && (
+        <ConfirmTierDeleteModal
+          isDeleting={isDeleting}
+          onClose={() => setIsConfirmingDelete(false)}
+          onConfirmDelete={onConfirmDelete}
+        />
+      )}
     </React.Fragment>
   );
 }
+
+function ConfirmTierDeleteModal({ isDeleting, onClose, onConfirmDelete }) {
```

Comment:
I think we can move this one to a separate component.
Suggested location: `components/edit-collective/tiers/ConfirmTierDeleteModal`.